static/app/views/insights/common/components/insightsTimeSeriesWidget.tsx

```diff
@@ -78,6 +85,8 @@ export function InsightsTimeSeriesWidget(props: InsightsTimeSeriesWidgetProps) {
             ? Area
             : Bars;
 
+      yAxes.push(timeSeries.yAxis);
+
```

Comment:
You might see some weird stuff here in places like Queues, I think we have a few spots where the `yAxis` of the series is something weird like `"published"` that isn't a real Y Axis that you can pass to Explore, so just watch out for that!

Reply:
Good call! i'll look into those as I implement this feature